packages/ui/src/tooltip/Tooltip.css

```diff
@@ -0,0 +1,24 @@
+.ui-tooltip-aria {
+  background-color: #252423;
```

Comment:
All colors should either come form the variables file or be added there. This looks to be --theme-gray-110